src/open_clip/hf_configs.py

```diff
@@ -20,5 +21,16 @@
       "layers": "num_hidden_layers",
     },
     "pooler": "mean_pooler",
-  }
+  },
+  # https://huggingface.co/docs/transformers/model_doc/mt5#mt5
+  "mt5": {
+    "config_names": {
+        "context_length": "relative_attention_max_distance", # very unsure of this, couldn't find anything better
```

Comment:
couldn't find anything closer, does mt5 not have a context length?

Reply:
looks a bit suspicious indeed

Reply:
fixed